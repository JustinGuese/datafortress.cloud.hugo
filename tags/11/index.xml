<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>11 on Datafortress.cloud</title><link>https://datafortress.cloud/tags/11/</link><description>Recent content in 11 on Datafortress.cloud</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Fri, 25 Jun 2021 12:40:24 +0600</lastBuildDate><atom:link href="https://datafortress.cloud/tags/11/index.xml" rel="self" type="application/rss+xml"/><item><title>Tech NEWS 2021-06-25</title><link>https://datafortress.cloud/blog/tech-news-2021-06-25/</link><pubDate>Fri, 25 Jun 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-06-25/</guid><description>Microsoft is bringing Android apps to Windows 11 with Amazonâ€™s Appstore (2 minute read) Windows 11 will be able to run Android apps natively.</description></item><item><title>Tech NEWS 2021-06-16</title><link>https://datafortress.cloud/blog/tech-news-2021-06-16/</link><pubDate>Wed, 16 Jun 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-06-16/</guid><description>Windows 11 leak reveals new UI, Start menu, and more (3 minute read) An early version of Windows 11 has leaked online ahead of its unveiling on June 24.</description></item></channel></rss>