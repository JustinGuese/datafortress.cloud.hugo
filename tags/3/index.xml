<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>3% on Datafortress.cloud</title><link>https://datafortress.cloud/tags/3/</link><description>Recent content in 3% on Datafortress.cloud</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Tue, 20 Jul 2021 12:40:24 +0600</lastBuildDate><atom:link href="https://datafortress.cloud/tags/3/index.xml" rel="self" type="application/rss+xml"/><item><title>Tech NEWS 2021-07-20</title><link>https://datafortress.cloud/blog/tech-news-2021-07-20/</link><pubDate>Tue, 20 Jul 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-07-20/</guid><description>Peloton plans to launch an in-app video game where you pedal to control a rolling wheel (2 minute read) Lanebreak is Peloton&amp;rsquo;s new in-app video game that involves riders changing their cadence and resistance to meet goals and control an on-screen rolling wheel.</description></item><item><title>Tech NEWS 2021-03-22</title><link>https://datafortress.cloud/blog/tech-news-2021-03-22/</link><pubDate>Mon, 22 Mar 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-03-22/</guid><description>This tiny car beat Teslaâ€™s Model 3 in January and February (2 minute read) The Hong Guang Mini EV outsold Tesla&amp;rsquo;s Model 3 in January and February.</description></item><item><title>Tech NEWS 2020-10-19</title><link>https://datafortress.cloud/blog/tech-news-2020-10-19/</link><pubDate>Mon, 19 Oct 2020 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2020-10-19/</guid><description>Nokia wins NASA contract to put a 4G network on the moon (1 minute read) Nokia has been awarded a $14.</description></item></channel></rss>