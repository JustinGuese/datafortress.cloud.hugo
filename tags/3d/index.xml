<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>3D on Datafortress.cloud</title><link>https://datafortress.cloud/tags/3d/</link><description>Recent content in 3D on Datafortress.cloud</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Mon, 17 Jan 2022 12:40:24 +0600</lastBuildDate><atom:link href="https://datafortress.cloud/tags/3d/index.xml" rel="self" type="application/rss+xml"/><item><title>Tech NEWS 2022-01-17</title><link>https://datafortress.cloud/blog/tech-news-2022-01-17/</link><pubDate>Mon, 17 Jan 2022 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2022-01-17/</guid><description>Nvidia’s AI-powered scaling makes old games look better without a huge performance hit (3 minute read) Nvidia&amp;rsquo;s latest game-ready driver includes a tool called Deep Learning Dynamic Super Resolution (DLDSR) that can make most games look sharper.</description></item><item><title>Tech NEWS 2021-07-07</title><link>https://datafortress.cloud/blog/tech-news-2021-07-07/</link><pubDate>Wed, 07 Jul 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-07-07/</guid><description>Amazon shifts Lumberyard to open source 3D game engine supported by 20 companies (13 minute read) Amazon has open-sourced its Lumberyard game engine and renamed it the Open 3D Engine.</description></item><item><title>Tech NEWS 2021-05-28</title><link>https://datafortress.cloud/blog/tech-news-2021-05-28/</link><pubDate>Fri, 28 May 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-05-28/</guid><description>Amazon Introduces Tiny ‘ZenBooths’ for Stressed Out Warehouse Workers (3 minute read) Amazon is introducing tiny booths for its workers to momentarily escape from their jobs.</description></item><item><title>Tech NEWS 2021-03-09</title><link>https://datafortress.cloud/blog/tech-news-2021-03-09/</link><pubDate>Tue, 09 Mar 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-03-09/</guid><description>SpaceX moves to beam Starlink internet into cars, boats, and aircraft (3 minute read) SpaceX is seeking regulatory approval to connect its Starlink network to cars, trucks, shipping boats, and aircraft.</description></item><item><title>Tech NEWS 2020-12-30</title><link>https://datafortress.cloud/blog/tech-news-2020-12-30/</link><pubDate>Wed, 30 Dec 2020 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2020-12-30/</guid><description>Apple Watch leak offers rare glimpse inside ‘Ultra security program’ (2 minute read) A video purporting to show an Apple Watch prototype has leaked.</description></item><item><title>How to integrate 3D models into websites and Quick View for Android and iOS devices</title><link>https://datafortress.cloud/blog/3d-model-integration/</link><pubDate>Thu, 24 Sep 2020 11:10:07 +0600</pubDate><guid>https://datafortress.cloud/blog/3d-model-integration/</guid><description>Integrating a 3D model into a website using Quick View for Android and iOS Try it out: Rotate the 3D object below, or look at it in AR using your phone or even VR if a VR-glass is connected to your computer.</description></item><item><title>Tech NEWS 2020-09-23</title><link>https://datafortress.cloud/blog/tech-news-2020-09-23/</link><pubDate>Wed, 23 Sep 2020 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2020-09-23/</guid><description>Here are Tesla’s biggest announcements from Battery Day (3 minute read) Elon Musk made some big announcements at Tesla&amp;rsquo;s Battery Day event.</description></item><item><title>Tech NEWS 2020-07-20</title><link>https://datafortress.cloud/blog/tech-news-2020-07-20/</link><pubDate>Mon, 20 Jul 2020 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2020-07-20/</guid><description>Leaked Gmail redesign integrates Chat, Meet, and document collaboration in one place (2 minute read) Google has announced a redesign for Gmail.</description></item></channel></rss>