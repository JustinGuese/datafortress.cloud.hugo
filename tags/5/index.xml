<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>5 on Datafortress.cloud</title><link>https://datafortress.cloud/tags/5/</link><description>Recent content in 5 on Datafortress.cloud</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Thu, 03 Dec 2020 12:40:24 +0600</lastBuildDate><atom:link href="https://datafortress.cloud/tags/5/index.xml" rel="self" type="application/rss+xml"/><item><title>Tech NEWS 2020-12-03</title><link>https://datafortress.cloud/blog/tech-news-2020-12-03/</link><pubDate>Thu, 03 Dec 2020 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2020-12-03/</guid><description>Leaked Galaxy Buds Pro image reveals new rounded design (1 minute read) An image of Samsung&amp;rsquo;s upcoming Galaxy Buds Pro true wireless earbuds has leaked, featuring a more rounded shape than the Galaxy Buds and Galaxy Buds Plus.</description></item><item><title>Tech NEWS 2020-10-01</title><link>https://datafortress.cloud/blog/tech-news-2020-10-01/</link><pubDate>Thu, 01 Oct 2020 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2020-10-01/</guid><description>Google announces the Pixel 5 for $699 (3 minute read) The Google Pixel 5 will be available in eight countries on October 15th, then in the US on October 29th for $699.</description></item></channel></rss>