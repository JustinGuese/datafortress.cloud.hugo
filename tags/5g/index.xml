<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>5g on Datafortress.cloud</title><link>https://datafortress.cloud/tags/5g/</link><description>Recent content in 5g on Datafortress.cloud</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Thu, 10 Jun 2021 12:40:24 +0600</lastBuildDate><atom:link href="https://datafortress.cloud/tags/5g/index.xml" rel="self" type="application/rss+xml"/><item><title>Tech NEWS 2021-06-10</title><link>https://datafortress.cloud/blog/tech-news-2021-06-10/</link><pubDate>Thu, 10 Jun 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-06-10/</guid><description>Facebook plans first smartwatch for next summer with two cameras, heart rate monitor (3 minute read) Facebook&amp;rsquo;s smartwatch will feature two cameras on a detachable display.</description></item><item><title>Tech NEWS 2021-03-31</title><link>https://datafortress.cloud/blog/tech-news-2021-03-31/</link><pubDate>Wed, 31 Mar 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-03-31/</guid><description>Snap Plans Hardware Push With AR Spectacles, Drone (1 minute read) Snap&amp;rsquo;s next-gen Spectacles will be able to layer Snapchat lenses onto the surrounding environment without using a smartphone&amp;rsquo;s camera.</description></item><item><title>Tech NEWS 2021-03-24</title><link>https://datafortress.cloud/blog/tech-news-2021-03-24/</link><pubDate>Wed, 24 Mar 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-03-24/</guid><description>Microsoft in talks with Discord over $10 billion-plus acquisition (1 minute read) Microsoft is in discussions to acquire Discord for more than $10 billion.</description></item><item><title>Tech NEWS 2020-07-16</title><link>https://datafortress.cloud/blog/tech-news-2020-07-16/</link><pubDate>Thu, 16 Jul 2020 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2020-07-16/</guid><description>Barack Obama, Joe Biden, Elon Musk, Apple, and others hacked in unprecedented Twitter attack (7 minute read) Twitter accounts of some major companies and individuals were compromised in an attack that resulted in Twitter blocking new tweets from all verified users and locking all compromised accounts.</description></item></channel></rss>