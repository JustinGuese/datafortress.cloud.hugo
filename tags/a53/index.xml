<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>a53 on Datafortress.cloud</title><link>https://datafortress.cloud/tags/a53/</link><description>Recent content in a53 on Datafortress.cloud</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Wed, 16 Mar 2022 12:40:24 +0600</lastBuildDate><atom:link href="https://datafortress.cloud/tags/a53/index.xml" rel="self" type="application/rss+xml"/><item><title>Tech NEWS 2022-03-16</title><link>https://datafortress.cloud/blog/tech-news-2022-03-16/</link><pubDate>Wed, 16 Mar 2022 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2022-03-16/</guid><description>Samsung’s Galaxy A33 leaks in full before its ‘Awesome’ launch (3 minute read) Samsung is set to announce three devices at its Awesome Galaxy A Event this week.</description></item></channel></rss>