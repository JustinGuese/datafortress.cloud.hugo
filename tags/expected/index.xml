<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>expected on Datafortress.cloud</title><link>https://datafortress.cloud/tags/expected/</link><description>Recent content in expected on Datafortress.cloud</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Wed, 08 Sep 2021 12:40:24 +0600</lastBuildDate><atom:link href="https://datafortress.cloud/tags/expected/index.xml" rel="self" type="application/rss+xml"/><item><title>Tech NEWS 2021-09-08</title><link>https://datafortress.cloud/blog/tech-news-2021-09-08/</link><pubDate>Wed, 08 Sep 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-09-08/</guid><description>Apple Event Announced: &amp;lsquo;California Streaming&amp;rsquo; on September 14 With iPhone 13, Apple Watch Series 7 Expected (3 minute read) Apple&amp;rsquo;s next event will be on September 14 at 10 am.</description></item><item><title>Tech NEWS 2021-03-25</title><link>https://datafortress.cloud/blog/tech-news-2021-03-25/</link><pubDate>Thu, 25 Mar 2021 12:40:24 +0600</pubDate><guid>https://datafortress.cloud/blog/tech-news-2021-03-25/</guid><description>Pfizer trials new pill that could stop early-stage Covid in its tracks (3 minute read) Pfizer has started stage one clinical trials on an oral pill designed to be an early intervention for Covid-19.</description></item></channel></rss>